app/forms/error_messages.py

```diff
@@ -54,6 +55,9 @@
         "Enter a 'period to' date later than the 'period from' date"
     ),
     "INVALID_DURATION": lazy_gettext("Enter a valid duration"),
+    "INVALID_MOBILE_NUMBER": lazy_gettext(
+        "Enter a UK mobile number in a valid format, for example, 07700 912345 or +44 7700 912345"
```

Comment:
Where did this number come from? A google search says it might belong to someone 😬 

We should pick one in the range 07700 900000 to 900999 (from https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama) to ensure we don't accidentally use a real phone number.

Reply:
The card